### PR TITLE
Update letsencrypt blog

### PR DESCRIPTION
**Pull request by MartinHjelmare**
* Use certbot with --standalone option instead of letsencrypt-auto.
* Update port instructions. Only port 80 is needed during challenge.

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
I didn't update the date. Should we write somewhere that the page was updated?

**Comment by Landrash** [comment]
@MartinHjelmare As someone who follows the blog I would appreciate there being at least a reference in next release post mentioning that the blog has been updated. 
Also writing something that the post has been updated gives confidence for those who read it for instructions that it's not outdated.

**Comment by MartinHjelmare** [comment]
Yes I agree. If I change the date in the front matter, I guess the post will be sorted to the front of the list of blog posts? Or is this dependent on the file name? Would that be a good solution?

**Comment by balloob** [comment]
No, we shouldn't change the url that this post is being served on. Instead just add a note after the introduction when and why it was updated

File: source/_posts/2015-12-13-setup-encryption-using-lets-encrypt.markdown [hunk 1]
```diff
@@ -46,48 +46,45 @@ First step is to acquire and set up our domain name. For this, go to [DuckDNS],
...
-IMPORTANT NOTES:
- - Congratulations! Your certificate and chain have been saved at
-   /etc/letsencrypt/live/hass-example.duckdns.org/fullchain.pem. Your cert
-   will expire on 2016-03-12. To obtain a new version of the
-   certificate in the future, simply run Let's Encrypt again.
- - If like Let's Encrypt, please consider supporting our work by:
-
-   Donating to ISRG / Let's Encrypt:   https://letsencrypt.org/donate
-   Donating to EFF:                    https://eff.org/donate-le
+$ echo 'deb http://ftp.debian.org/debian jessie-backports main' | sudo tee --append /etc/apt/sources.list.d/backports.list
```
Comment (balloob) [inline source/_posts/2015-12-13-setup-encryption-using-lets-encrypt.markdown]: I would love to keep it platform independent like it was before. What is wrong with git cloning the repo? (or can we pip install it?)

Comment (MartinHjelmare) [inline source/_posts/2015-12-13-setup-encryption-using-lets-encrypt.markdown]: I'm following the recommended install procedure. Also personally I would never suggest to someone that they should git clone and install, as the first resort, if there is a maintained apt or similar package for their platform. Installing a package from the OS package manager is usually the simplest and most error less way, and updates will be seamless.

I can add a link to the docs where there are instructions for all platforms.

You're call though.

Comment (MartinHjelmare) [inline source/_posts/2015-12-13-setup-encryption-using-lets-encrypt.markdown]: I can test if pip install is working without additional configuration. That could be an option in that case.

Comment (MartinHjelmare) [inline source/_posts/2015-12-13-setup-encryption-using-lets-encrypt.markdown]: What the heck, I'll download and run the platform independent script instead. It's not so much more effort. I'll link to the docs and let users follow whatever install method they like.

Thanks for input.

**Comment by balloob** [comment]
Looks good! :dolphin:

**Pull request merged**